.github/workflows/ci_cd.yml

```diff
@@ -93,6 +93,19 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
 
+  check-vulnerabilities:
+    name: "Check library vulnerabilities (default mode - only on main)"
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check library vulnerabilities with default mode"
+        uses: ansys/actions/check-vulnerabilities@v6.0
+        with:
+          python-version: ${{ env.MAIN_PYTHON_VERSION }}
+          token: ${{ secrets.PYANSYS_CI_BOT_TOKEN }}
+          python-package-name: 'ansys-<product>-<library>'
+
+
```

Comment:
Hey.. you need to adapt things here 😄 

You will also need to add the dev version

Reply:
Yes, I noticed it... I pushed my commit a little bit too fast. Thanks for checking it! 😄